locales/en/translation.json

```diff
@@ -4,6 +4,7 @@
     "export-gist": "Export Gist",
     "export-repo": "Export Repo",
     "share-to-classroom": "Share To Classroom",
+    "add-or-edit-instructions": "Add/Edit Instructions",
```

Comment:
I prefer having "Add Instructions" and "Edit Instructions" as two different strings. And have these strings show up whether or not instructions are present in the Popcode project.